### PR TITLE
netty: update to 4.1.17-Final

### PR DESCRIPTION
**Pull request by carl-mastrangelo**

### CODE REVIEW

**Comment by ejona86** [comment]
Can you also update that table in SECURITY.md?

**Comment by carl-mastrangelo** [comment]
@ejona86 done

File: SECURITY.md [hunk 1]
```diff
@@ -326,7 +326,8 @@ grpc-netty version | netty-code-http2 version | netty-tcnative-boringssl-static
 1.4.x              | 4.1.11.Final             | 2.0.1.Final
 1.5.x              | 4.1.12.Final             | 2.0.5.Final
 1.6.x              | 4.1.14.Final             | 2.0.5.Final
-1.7.x-             | 4.1.16.Final             | 2.0.6.Final
+1.7.x              | 4.1.16.Final             | 2.0.6.Final
```
Comment (ejona86) [inline SECURITY.md]: 1.7.x-1.8.x

Comment (carl-mastrangelo) [reply SECURITY.md]: Also done

**Pull request merged**